packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.controller.js

```diff
@@ -43,6 +52,7 @@ export default class newDnsZoneCtrl {
 
     this.WORKFLOW_TYPE = workflowConstants.WORKFLOW_TYPES.ORDER;
     this.PRICING_TYPE = pricingConstants.PRICING_CAPACITIES.RENEW;
+    this.getAlerterId = newDnsZoneCtrl.getAlerterId;
```

Comment:
This line could be removed if you chose to implement https://github.com/ovh/manager/pull/9613#discussion_r1267577878